Sources/Components/ReviewButtonView/ReviewButtonView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+public protocol ReviewButtonViewDelegate: AnyObject {
+    func reviewButtonView(_ reviewButtonView: ReviewButtonView, giveReviewWasTapped startReview: Bool)
+}
+
+public final class ReviewButtonView: UIView {
+    // MARK: - Public
+
+    public weak var delegate: ReviewButtonViewDelegate?
+    public var buttonText: String = "" {
+        didSet {
+            reviewButtonControl.text = buttonText
+            accessibilityLabel = buttonText
+        }
+    }
+
+    // MARK: - Private
+
+    private let height: CGFloat = 64
+
+    private lazy var hairlineSeperator: UIView = {
+        let view = UIView(withAutoLayout: true)
+        view.backgroundColor = .sardine
+        return view
+    }()
+
+    private lazy var reviewButtonControl: ReviewButtonControl = {
+        let control = ReviewButtonControl()
+        control.delegate = self
+        return control
+    }()
+
+    // MARK: - Init
+
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+
+    // MARK: - Overrides
+
+    public override func didMoveToSuperview() {
+        super.didMoveToSuperview()
```

Comment:
Very minor nitpick, but you don't really need to call the super-class' implementation, as the default implementation does nothing.

Reply:
Noted and fixed👍